package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "barefoot",
+  "version": "1.0.0",
+  "description": "Develop Barefoot nomad frontend",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack-dev-server --mode development --hot --open --config webpack.config.js",
+    "build": "webpack --mode production",
+    "start": "HTTPS=true serve -s ./dist"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/andela/team-odd-bn-frontend"
+  },
+  "author": "Eric Niyongabo",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/andela/team-odd-bn-frontend/issues"
+  },
+  "homepage": "https://github.com/andela/team-odd-bn-frontend#readme",
+  "dependencies": {
+    "@babel/core": "^7.7.5",
+    "@babel/preset-env": "^7.7.6",
+    "@babel/preset-react": "^7.7.4",
```

Comment:
I have a question concerning babel, is there need to have it in the dependencies yet the application we run in heroku is already build, I mean is already converted to es5

Reply:
You are right @hezronkimutai.